dataprofiler/profilers/numerical_column_stats.py

```diff
@@ -405,7 +405,7 @@ def mean(self) -> float:
         """Return mean value."""
         if self.match_count == 0:
             return 0
-        return float(self.sum) / self.match_count
+        return float(self.sum) / float(self.match_count)
```

Comment:
mypy compains if self.match_count isn't a float?

Reply:
Actually, now that I look at the code again, I don't see `self.match_count` getting initialized anywhere. This would explain why it's being typed as `Any` (thus requiring some sort of cast).

Reply:
I fixed it by manually adding a type for it in the constructor. I assume `self.match_count` is populated in some dynamic way so mypy isn't able to pick up its type automatically.